.github/workflows/pr-pull.yaml

```diff
@@ -14,13 +14,13 @@ concurrency:
   cancel-in-progress: true
 jobs:
   check-format:
-    name: Check Formatting 🔍
+    name: Check Format 🔍
     uses: ./.github/workflows/check-format.yaml
     permissions:
       contents: read
 
   build-project:
-    name: Build Project 🧱
+    name: BUILD 🧱
```

Comment:
I don't think this change is meaningful, as the first token is shortened to a single letter by GitHub anyway.

Reply:
See https://github.com/obsproject/obs-studio/pull/9289#issuecomment-1646612797

Reply:
I specifically did not use abbreviations and capital case names in the rework, so I'd prefer "Build" there (removing "Project" isn't detrimental IMO).

I do prefer keeping "Pull Request" though and while I don't wish to diminish the view on mobile phones, the overall GitHub UI is not really usable in portrait mode (and the site isn't really optimised for that anyway).

Reply:
I concur with @PatTheMav . Use "Build".

Reply:
Looks like this has already been changed. Resolving.